### PR TITLE
Deflake a test; catch unexpected diagnostics.

### PR DESCRIPTION
**Pull request by rictic**
- [x] CHANGELOG.md not updated, internal change

---
This change is [Reviewable](https://reviewable.io/reviews/polymer/polymer-editor-service/111)

### CODE REVIEW

**Comment by rictic** [comment]
I _think_ this will solve our flaky test problem.

File: src/test/util.ts [hunk 1]
```diff
@@ -141,6 +150,22 @@ export class TestClient {
     });
   }
 
+  async cleanup() {
+    await new Promise((resolve) => setTimeout(resolve, 1));
```
Comment (aomarks) [inline src/test/util.ts]: Comment for this function? And why we need the timeout.

Comment (rictic) [reply src/test/util.ts]: Done

**Comment by rictic** [comment]
(note: looks like this did! went from at least one flake in most travis runs (6 test runs per PR) to zero!)

**Pull request merged**